mavros/src/plugins/global_position.cpp

```diff
@@ -118,11 +118,12 @@ class GlobalPositionPlugin : public MavRosPlugin {
 		// fill GPS status fields using GPS_RAW data
 		auto raw_fix = uas->get_gps_fix();
 
-		gp_fix->status.service = raw_fix->status.service;
-		gp_fix->status.status = raw_fix->status.status;
-
-		gp_fix->position_covariance = raw_fix->position_covariance;
-		gp_fix->position_covariance_type = raw_fix->position_covariance_type;
+		if (raw_fix != nullptr) {
+			gp_fix->status.service = raw_fix->status.service;
+			gp_fix->status.status = raw_fix->status.status;
+			gp_fix->position_covariance = raw_fix->position_covariance;
+			gp_fix->position_covariance_type = raw_fix->position_covariance_type;
+		}
 
```

Comment:
Please add `else` block that fill those fields as unknown, e.g. cov `1, 0...`, type unknown etc.